deploy.cmd

```diff
@@ -0,0 +1,143 @@
+@if "%SCM_TRACE_LEVEL%" NEQ "4" @echo off

```

Comment:
what is the purpose of this deployment script?
Don't we use default deployment pipeline?

Reply:
See above comment.

This is the default deploy script with the func pack check removed. This allows us to ignore funcpack even on existing collectors.